_sections/home-english.md

```diff
@@ -170,6 +170,12 @@ In general, choice names should be unique within a single-choice list. If two ch
 | ===========             |
 | settings                |
 
+#### Cascading selects
+
+It's common to have multiple connected select questions where earlier ones filter the options for later ones. For example, a form might start out by asking the respondent's state, then their district, and finally their city. An alternative would be to directly have the user select their city name from a list of all cities, but the city list would be very long and wouldn't work for city names that are repeated in different districts or states.
```

Comment:
> If you want question responses to filter the options available in later questions, use a cascading select. For example, if you want to ask which city the respondent lives in, instead of choosing from all cities in the world, you could first ask for their country, state (in that country), district (in that state), then city (in that district).